src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/PersonPictureTests/PersonPictureLateBindingPage.xaml.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
Is this ported from WinUI? I can't find it in the sources (https://github.com/search?q=repo%3Amicrosoft%2Fmicrosoft-ui-xaml%20PersonPictureLateBindingPage&type=code)

Reply:
Oh this is the test! Good catch, No, it is not ported from WinUI. I copied another PersonPicture test and it seems I forgot to delete this part. Deleting it now.